data/dump/count_info.data.R

```diff
@@ -1,5 +1,7 @@
 N <- 259
 P <- 18
+str <- 
+c(1, 1, 1, 1, 1, 10, 1, 10, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1)
 end <- 
 c(259, 259, 259, 259, 259, 250, 259, 250, 259, 259, 259, 259, 259, 259,
```

Comment:
Ok, so there are a few things that are weird. Some wires got crossed in the data file and I'm fixing it now.